evalai/main.py

```diff
@@ -31,6 +34,16 @@ def main(ctx):
             " for challenge_id\nand phase_id of the challenges and phases."
         )
         echo(welcome_text)
+    latest_version = get_latest_version()
+    if __version__ < latest_version:
+        echo(
+            style(
+                    "\nUpdate:\n"
+                    "\nPlease install the latest version of EvalAI-CLI!\n",
```

Comment:
Please add the `pip` command for installing latest EVALAI-CLI version in the message.